application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -44,6 +45,8 @@ private Config(@JsonProperty(value = "token", required = true) String token,
             @JsonProperty(value = "discordGuildInvite", required = true) String discordGuildInvite,
             @JsonProperty(value = "modAuditLogChannelPattern",
                     required = true) String modAuditLogChannelPattern,
+            @JsonProperty(value = "modMailChannelPattern",
+                    required = true) String modMailChannelPattern,
```

Comment:
u'll need to add this in `config.json.template` too

Reply:
Done